examples/purchaseTesterTypescript/android/build.gradle

```diff
@@ -19,3 +19,11 @@ buildscript {
         classpath("com.facebook.react:react-native-gradle-plugin")
     }
 }
+
+allprojects {
+  repositories {
+    google()
+    mavenCentral()
+    mavenLocal()
```

Comment:
Should we also add `mavenLocal` above? (In the buildscript > repositories section